frontend-react/src/styles/_reportstream.scss

```diff
@@ -110,6 +110,11 @@
   border-color: #dfe1e2;
 }
 
+.record-count {
```

Comment:
Is there a way we can handle this via the uswds sass utility classes instead of our own CSS? https://designsystem.digital.gov/utilities/font-size-and-family/